static/js/jira-manual-app-creation.js

```diff
@@ -28,7 +28,7 @@ AJS.$("#jiraManualAppCreation__form").on("aui-valid-submit", (event) => {
   // Reading the content of the file
   const file = $("#privateKeyFile")[0].files[0];
   const reader = new FileReader();
-  reader.readAsDataURL(file);
+  reader.readAsText(file);
```

Comment:
Just wonder why do we want to read it from client side? Is it to verify the file exist?
If read fail, what's the UX like?

Reply:
We are reading the file contents from the client side to avoid having to upload anything in the server. 
The failure is not yet handled, if it fails it should error out in console. The UX only shows error for invalid files(extensions that are not `pem`).

Reply:
`to avoid having to upload anything` <-- I don't get it? Aren't we supposed to upload the clientKey to server to save it?

Reply:
We are not uploading any files, if that makes sense. From the pem file added in the `input` field, we simply pull the contents and save that as the `privateKey`.